powerspy.py

```diff
@@ -410,8 +418,9 @@ def exit_gracefully(signal, frame):
   #parser.add_argument('--version', action='version', version='%(prog)s unreleased')
   args = parser.parse_args()
 
-  if args.verbose > 0:
-    logging.basicConfig(level=logging.DEBUG)
+  if args.verbose:
```

Comment:
Revert this.

Reply:
In Python 3, we can't compare an int with an undefined type. I got this error when I run the script without adding ```-v``` : ```TypeError: '>' not supported between instances of 'NoneType' and 'int'```
So I added this check first. But there might be a more elegant way to do it maybe?

Reply:
in `add_argument` for verbose, let's add "default=0".

Reply:
Done.